From 55ae454038aed1f18e12e336a58a8cc03b0ee0a9 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 9 Apr 2024 16:36:00 +0800
Subject: [PATCH 1/5] AOSCOS: fix(window.style): set default window width to
 360px

This allows the Telegram window to scale properly on the PinePhone.
---
 Telegram/SourceFiles/window/window.style | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Telegram/SourceFiles/window/window.style b/Telegram/SourceFiles/window/window.style
index fa6a607522..a0a4412d67 100644
--- a/Telegram/SourceFiles/window/window.style
+++ b/Telegram/SourceFiles/window/window.style
@@ -10,7 +10,7 @@ using "ui/widgets/widgets.style";
 using "chat_helpers/chat_helpers.style";
 using "boxes/boxes.style"; // UserpicButton
 
-windowMinWidth: 380px;
+windowMinWidth: 360px;
 windowMinHeight: 480px;
 windowDefaultWidth: 800px;
 windowDefaultHeight: 600px;
@@ -19,7 +19,7 @@ windowBigDefaultHeight: 768px;
 
 columnMinimalWidthLeft: 260px;
 columnMaximalWidthLeft: 540px;
-columnMinimalWidthMain: 380px;
+columnMinimalWidthMain: 360px;
 columnMinimalWidthThird: 292px;
 columnMaximalWidthThird: 1176px;
 
-- 
2.55.0

